docs/technical_reference/unit_models/reverse_osmosis_0D.rst

```diff
@@ -174,12 +176,13 @@ Equations
    "Membrane area (spiral-wound)",":math:`A_m = 2LW`"
    "Pressure drop",":math:`ΔP = (\frac{ΔP}{Δx})_{avg}L`"
    "Feed-channel velocity",":math:`v_f = Q_f/A_c`"
-   "Friction factor",":math:`f = 0.42+\frac{189.3}{Re}`"
+   "Feed-channel volume", ":math:`V_{f} = LWh_{ch}`"
+   "Friction factor (flat-plate)",":math:`f = 0.42+\frac{189.3}{Re}`"
+   "Friction factor (spiral-wound)",":math:`f = \frac{6.23}{Re^{0.3}}`"
```

Comment:
@fahim831 good catch- I revised this and the exponent should show up as anticipated now in the documentation.

Reply:
Here's what it looks like now:
![image](https://github.com/user-attachments/assets/d3ae5dc2-729a-40df-8fef-b035c4318aff)

Reply:
Note-- this isn't related to the mini-step I took towards enabling a dynamic RO model. I just realized that this was missing from our RO documentation in the process of adding channel volume to the documentation.